CMakeLists.txt

```diff
@@ -183,6 +183,7 @@ message(STATUS "Build with HIP ${hip_VERSION}")
 target_flags(HIP_COMPILER_FLAGS hip::device)
 # Remove cuda arch flags
 string(REGEX REPLACE --cuda-gpu-arch=[a-z0-9]+ "" HIP_COMPILER_FLAGS "${HIP_COMPILER_FLAGS}")
+string(REGEX REPLACE --offload-arch=[a-z0-9]+ "" HIP_COMPILER_FLAGS "${HIP_COMPILER_FLAGS}")
```

Comment:
Could you please explain why we need this.

Reply:
@pfultz2 can you comment?

Reply:
hip::device was changed to use `--offload-arch` flag instead of `--cuda-gpu-arch`. If we dont remove these flags, MIOpen will try to compile for an arch different from the current device.